packages/docusaurus-theme-classic/src/theme/CodeInline/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import clsx from 'clsx';
+import {usePrismTheme, useThemeConfig} from '@docusaurus/theme-common';
+import Highlight, {defaultProps, type Language} from 'prism-react-renderer';
+import type {Props} from '@theme/CodeInline';
+
+export default function CodeInline({
+  language: languageProp,
+  className = '',
+  children,
+}: Props) {
+  const {
+    prism: {defaultLanguage},
+  } = useThemeConfig();
+  const language = languageProp ?? defaultLanguage ?? 'text';
+  const prismTheme = usePrismTheme();
+
+  return (
+    <code className={clsx(className, `language-${language}`)}>
+      <Highlight
+        {...defaultProps}
+        theme={prismTheme}
+        code={children}
+        language={language as Language}>
+        {({tokens, getTokenProps}) => {
+          if (tokens.length !== 1) {
+            // This is actually multi-line (or empty) code. Multi-line _should_ never happen. Just return the code without highlighting I guess?
+            return children;
```

Comment:
I am not sure if there is a better way to handle this, but I guess it is sufficient.